.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,40 @@
+name: Enhancement Request
+description: Suggest an enhancement to Uno Platform
+labels: [kind/enhancement, triage/untriaged, difficulty/tbd]
+body:
+  - type: textarea
+    id: enhancement-description
+    attributes:
+      label: What would you like to be added
+      description: Describe in detail what is the feature or functionality you are missing
+
+  - type: textarea
+    id: enhancement-use-case
+    attributes:
+      label: Why is this needed
+      description: Describe the use case of this enhancement
+
+  - type: dropdown
+    id: enhancement-platforms
+    attributes:
+      label: For which platform
+      multiple: true
+      options:
+        - WebAssembly
+        - Android
+        - iOS
+        - macOS (AppKit)
+        - Mac Catalyst
+        - Skia (WPF)
+        - Skia (GTK on Linux/macOS/Windows)
+        - Skia (Linux Framebuffer)
+        - Skia (Tizen)
+        - Windows
+        - Build tasks
+        - Solution Templates
```

Comment:
I assume these should go to Uno.Templates?

Reply:
True! Will remove the entry

Reply:
You can also add an entry here:

https://github.com/unoplatform/uno/blob/73fb9498a10fb074b7b4adba493312ee4bb31d23/.github/ISSUE_TEMPLATE/config.yml that routes users to the Uno.Templates repo